state-chain/pallets/cf-emissions/src/lib.rs

```diff
@@ -291,8 +293,8 @@ impl<T: Config> Pallet<T> {
 			)
 		}) {
 			Ok(ScheduledEgressDetails { egress_id, egress_amount, .. }) => {
-				T::Issuance::burn_offchain(egress_amount.into());
-				Self::deposit_event(Event::NetworkFeeBurned { amount: egress_amount, egress_id });
+				T::Issuance::burn_offchain(egress_amount.saturating_sub(flip_to_be_sent_to_gateway).into());
```

Comment:
nit: can deduplicate

```suggestion
    let flip_to_burn = egress_amount.saturating_sub(flip_to_be_sent_to_gateway);
	 T::Issuance::burn_offchain(flip_to_burn.into());
	 Self::deposit_event(Event::NetworkFeeBurned { amount: flip_to_burn, egress_id });
```